internal/providers/azure/network.go

```diff
@@ -0,0 +1,152 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azure
+
+import (
+	"context"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-01-01/network"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/banzaicloud/pipeline/secret"
+
+	intNetwork "github.com/banzaicloud/pipeline/internal/network"
+	pkgAzure "github.com/banzaicloud/pipeline/pkg/providers/azure"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+type azureNetwork struct {
+	cidr string
```

Comment:
Why not make it `[]string` since its a list of cidrs?

Reply:
If it was a `[]string` then either the getter would have to do the concatenation or the query interface would have to be changed to propagate the new return type (and that would break the API)